source/Components/Button.cpp

```diff
@@ -0,0 +1,328 @@
+/*
+** Project FileShare, 2022
+**
+** Author Léo Lhuile
+**
+** Started on  Sun Feb 13 22:03:32 2022 Léo Lhuile
+** Last update Mon Aug 29 20:45:51 2022 Léo Lhuile
+**
+** Button.cpp : Implementation of Button class
+*/
```

Comment:
Just realized all the headers are outdated 🙈 

There's an extension for that : https://marketplace.visualstudio.com/items?itemName=doi.fileheadercomment